docs/commands/kam_component_add.md

```diff
@@ -27,6 +27,8 @@ kam component add [flags]
   -h, --help                      help for add
       --interactive               If true, enable prompting for most options if not already specified on the command line
       --output string             Folder path to the Application to add the Component (default "./")
+      --route string              If you specify the route flag and pass the string, that string will be in the route.yaml that is generated
```

Comment:
For the description, use "The route to expose the component with. Referenced in generated route.yaml"